plugins/Microsoft.DocAsCode.Build.MemberLevelManagedReference/SplitClassPageToMemberLevel.cs

```diff
@@ -58,9 +59,39 @@ public override IEnumerable<FileModel> Prebuild(ImmutableList<FileModel> models,
                     else
                     {
                         treeMapping.Add(result.Uid, Tuple.Create(model.OriginalFileAndType, result.TreeItems));
-                        collection.AddRange(result.Models);
                     }
                 }
+                else
+                {
+                    AddModelToDict(model, modelsDict, dupeModelsDict);
+                }
+            }
+
+            // New dupe models
+            foreach (var dupe in dupeModelsDict)
+            {
+                var dupeModels = dupe.Value;
+
+                dupeModels.Add((modelsDict[dupe.Key]));
+                modelsDict.Remove(dupe.Key);
```

Comment:
Can `dupeModelsDict` be a simple `List<string>`? Currently it's a dictionary, whose key's only usage is here. I see if this is changed to list, `modelsDict.Remove` will be called multiple times for the same key, which I think is acceptable.